.circleci/config.yml

```
@@ -10,7 +10,14 @@ jobs:
       - run:
           name: "Build COMPASS docker image"
           command: |
-            docker build . -f docker/Dockerfile
+            docker build . -f docker/Dockerfile -t test_image
+      - run:
+          name: "Run integration tests"
+          command: |
+            docker run test_image bash -ci "cd ~/OPERA/COMPASS &&\
+                                            conda init bash &&\
+                                            conda activate COMPASS &&\
+                                            python -m pytest"
```

```suggestion
            docker run test_image python -m pytest
```
 In line of the suggestion in `Dockerfile` below.